### DISCUSSION TITLE
"At Wit's End" event issues

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15175] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

"At Wit's End" event has two issues
1. you need to press E twice to start the conversation
2. artie keeps following you if you decline him

### Reproduction steps

1. triggerevent atwitsend
2. decline artie
3. artie keeps following you

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this and found no issues.
Closing discussion as completed.
